lib/src/git.rs

```diff
@@ -1229,14 +1230,11 @@ fn render_ref_in_unexpected_location_details(refs: &[String]) -> String {
 #[derive(Clone, Debug)]
 pub struct GitBranchPushTargets {
     pub branch_updates: Vec<(String, BranchPushUpdate)>,
-    pub force_pushed_branches: HashSet<String>,
 }
 
 pub struct GitRefUpdate {
     pub qualified_name: String,
     pub expected_current_target: Option<CommitId>,
-    // Short-term TODO: Delete `force`
-    pub force: bool,
```

Comment:
I'm not pretty sure, but we might still need a "force" push for some integrations like gerrit.

@thoughtpolice do you have any idea?

Reply:
The plan is to always force push and do all the safety checks ourselves.

Reply:
> We end up always force pushing and doing all the safety checks ourselves.

Yeah, my concern was we might need a real "force" push (that doesn't respect remote refs at all) to implement some forge integrations, but I don't know. If there isn't an immediate need, we can restore it later.

Reply:
It's a good point that stranger git servers might send strange information to the push negotiation function or expect it to succeed in weird cases. We should hopefully be able to deal with it on a case-by-case basis and adjust the logic inside the push negotiation accordingly.

I don't know anybody I could ask to foresee this, perhaps Austin will have an idea. We could ask on the Git Discord, but I'm not sure how to phrase the question.